openmetadata-ui/src/main/resources/ui/src/styles/variables.less

```diff
@@ -34,3 +36,5 @@
 @padding-xlg: 48px;
 @margin-xlg: 48px;
 @card-shadow: 1px 1px 3px rgba(0, 0, 0, 0.12);
+@border-color: #d9ceee;
```

Comment:
This might override all the border-color do you intend to do so?

Reply:
Yes we are using `#d9ceee` color for border cc @devyani-kaushik